dm/pkg/storage/utils.go

```diff
@@ -142,15 +143,15 @@ func RemoveAll(ctx context.Context, dir string, storage bstorage.ExternalStorage
 
 	err = storage.WalkDir(ctx, &bstorage.WalkOption{}, func(filePath string, size int64) error {
 		err2 := storage.DeleteFile(ctx, filePath)
-		if errors.Cause(err2) == gstorage.ErrObjectNotExist {
+		if errors.Cause(err2) == gstorage.ErrObjectNotExist || bloberror.HasCode(err2, bloberror.BlobNotFound) {
```

Comment:
Can we only skip this error when this is az blob?  For other object storage, we can not skip this error.

Reply:
sure, we can ignore the corresponding errors by backend type.